libunwind/src/config.h

```diff
@@ -223,6 +221,41 @@
     } while (0)
 #endif
 
+#define _LIBUNWIND_TRACING_COMPACT_UNWIND (0)
+#if !_LIBUNWIND_TRACING_COMPACT_UNWIND
+#define _LIBUNWIND_TRACE_COMPACT_UNWIND0(msg)
+#define _LIBUNWIND_TRACE_COMPACT_UNWIND(msg, ...)
+#else
+#define _LIBUNWIND_TRACE_COMPACT_UNWIND0(msg)                                  \
+  do {                                                                         \
+    fprintf(stderr, msg);                                                      \
+  } while (0)
+#define _LIBUNWIND_TRACE_COMPACT_UNWIND(msg, ...)                              \
+  do {                                                                         \
+    fprintf(stderr, msg, __VA_ARGS__);                                         \
+  } while (0)
```

Comment:
```suggestion
#define _LIBUNWIND_TRACE_COMPACT_UNWIND(...)                                   \
  do {                                                                         \
    fprintf(stderr, __VA_ARGS__);                                         \
  } while (0)
```
Does this work?

Reply:
It probably would; the reason why I did it the way I did was consistency with the existing `_LIBUNWIND_LOG`/`_LIBUNWIND_LOG0` macros. I guess best would be to just rewrite those in your suggested way as well. But maybe this should be done in a separate PR to keep this one focused on just replacing `fprintf`?

Reply:
Sure fine by me :)